lib/pgcli/rails/tasks.rake

```diff
@@ -11,6 +15,8 @@ task :pgcli do
           ["--include-password"]
         end
 
-  console = Pgcli::Rails::DBConsole.new(opt)
```

Comment:
🗒️ Instead of sub-classing `DBConsole`, we now just apply a patch with `prepend`. The outcome is the same.